assets/js/modules/search-console/components/dashboard/DashboardImpressionsWidget.js

```diff
@@ -67,7 +67,7 @@ function DashboardImpressionsWidget() {
 
 	if ( error ) {
 		trackEvent( 'plugin_setup', 'search_console_error', error.message );
-		return getDataErrorComponent( __( 'Search Console', 'google-site-kit' ), error.message );
+		return getDataErrorComponent( 'search-console', error );
```

Comment:
See above, let's update this to `getDataErrorComponent( 'search-console', error.message, false, false, false, error )` so that everything is passed as expected.

Reply:
Updated.